Task_20/TestResult/Yandex.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="utf-8" standalone="no"?>
```

Comment:
I don't know if you're going to using such xml files in your next tasks, but there is potential vulnerability regarding absolute paths using. This file won't be work if your tests someone run on his local machine

Reply:
xml file added in .gitignore